modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -76,7 +76,7 @@ The following Technology Preview features are enabled by this feature set:
 ** `ServiceAccountTokenPodNodeInfo`
 ** `TranslateStreamCloseWebsocketRequests`
 ** `UpgradeStatus`
-** `VSphereMultiVCenters`
+** Multiple vCenter Support.  Enables {product-title} to support creating machines across multiple vCenter servers. (`VSphereMultiVCenters`)
```

Comment:
we are not explaining other features, so we can keep this consistent. Just curious to know if this field had already landed in previous release despite single vcenter support?

Reply:
This field may have been in earlier release when function was starting.  We can remove this if thats what we are going for now.  Please ignore this if needed.

Reply:
Thanks Neil for the clarification! We can get rid of the description here. We have a separate PR by one of the Storage docs writer explaining the feature. That would probably suffice. I will let the field stay as-is.